NeRF.py

```diff
@@ -216,23 +217,64 @@ def __init__(self, args, kernelsnet=None):
         self.output_ch = 5 if args.N_importance > 0 else 4
 
         skips = [4]
-        self.mlp_coarse = NeRF(
-            D=args.netdepth, W=args.netwidth,
-            input_ch=self.input_ch, output_ch=self.output_ch, skips=skips,
-            input_ch_views=self.input_ch_views, use_viewdirs=args.use_viewdirs)
-
-        self.mlp_fine = None
-        if args.N_importance > 0:
-            self.mlp_fine = NeRF(
-                D=args.netdepth_fine, W=args.netwidth_fine,
+        #Plenoxel option
+        if args.plenoxel:
```

Comment:
이 밑으로는 한글로 달게

조건문으로 할때 default setting 이 제일 위에 오는게 유지보수에 좋아
우리가 plenoxel을 수정할 수도 있으니까
```suggestion
        if args.architecture == 'nerf':

        elif args.architecture == 'plenoxel':
        
        else: 
            raise NotImplementedError
        
```